### DISCUSSION TITLE
Trees with Stripped logs

### DISCUSSION THREAD

**Discussion by vico93** [RakambdaOrg/FallingTree#687] [Q&A]
Is possible to tweak the mod to accept stripped logs (the ones you get when you right-click with an axe on a regular log) as a valid "felling" type of block?

Since barking is an extraordinary annoying feature and sometimes i ended up accidentally stripping a random log from the tree i want to fell, after doing that the tree doesnt fall anymore.

**Comment by Rakambda** [comment]
You can add them to the [allowed list of blocks](https://github.com/RakambdaOrg/FallingTree/wiki/Settings---Trees#:~:text=Description-,Allowed%20logs,-Allows%20you%20to) in the config, and turn on [mixed logs](https://github.com/RakambdaOrg/FallingTree/wiki/Settings---Trees#:~:text=Allow%20trunks%20with%20mixed%20log%20blocks).
